build.sbt

```diff
@@ -102,10 +110,9 @@ lazy val `support-frontend` = (project in file("support-frontend"))
   .aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`)
 
 lazy val `support-workers` = (project in file("support-workers"))
-  .enablePlugins(JavaAppPackaging, RiffRaffArtifact)
+  .enablePlugins(RiffRaffArtifact).disablePlugins(ReleasePlugin, SbtPgp, Sonatype)
```

Comment:
didn't actually need Java app packaging, as that is designed to add platform specific shell scripts, but all we need is a plain jar file.